salesagility/SuiteCRM
# Fix #10237 - Range validation in inline edition

- Closes #10237




## Description




The inline edition did not validate that the new value is within the range of a numerical field with a defined maximum and/or minimum.

The implemented solution has two parts:
- **controller.php**: fills in ```$validate_array``` the parameters for the range validation: the ```type``` as the validation type (```range```), ```max``` and ```min```
- **inlineEditing.js**: checks this validation type and for fields to validate as ```range```, call the defined function ```addToValidateRange```

## How To Test This

1. In Studio, create an integer field, with Min Value 1 and Max Value 15
2. Add the field to the detail and list view.
3. Verify that we can not add a value outside the defined range.


## Types of changes


### Final checklist
